lib/travis/build/addons/sonarqube.rb

```diff
@@ -0,0 +1,31 @@
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Sonarqube < Base
+        SUPER_USER_SAFE = true
+        TEMPLATES_PATH = File.expand_path('templates', __FILE__.sub('.rb', ''))
+        DEFAULT_SQ_HOST_URL = "https://nemo.sonarqube.org"
+        SCANNER_CLI_VERSION = "2.6.1"
+
+        def after_header
+          @scanner_home = "$HOME/.sonarscanner"
+          sh.raw template('sonarqube.sh', home: @scanner_home, version: SCANNER_CLI_VERSION )
```

Comment:
For a short script such as `sonarqube.sh`, it is better to define a method in the addon itself, rather than using a template to include it in the build script. It is easier to understand the flow of the addon that way.